.codecov.yml

```diff
@@ -0,0 +1,21 @@
+codecov:
+  notify:
+    after_n_builds: 1
+
+coverage:
+  status:
+    project:
+      default:
+        target: 70%
+        if_not_found: success

```

Comment:
What exactly is 'if_not_found' doing?

Reply:
This is taken from the [JS repo](https://github.com/aws-amplify/amplify-js/blob/4b5841984727794217295adb37f2f9f7fddb509e/codecov.yml#L12) 

https://docs.codecov.io/docs/commit-status#if_not_found

> success
: the status will pass if there is no report for the head. Use this on commits / PRs where you won't be uploading coverage but still want codecov status checks to pass. 

I guess it makes sense to start with this?